dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -308,6 +310,8 @@ export const Card = ({
 	);
 
 	const showQuotes = !!showQuotedHeadline;
+	const rightAlignFooter = isOnwardContent;
+	const roundedImage = isOnwardContent;
```

Comment:
hmm... my gut says its better to pass through the prop itself so its more explicit whats determining the changes rather than adding another layer of abstraction.

rightAlign *because* isOnwardContent
feels clearer to me than
rightAlign *because* roundedImage

Reply:
hmm... my gut says its better to pass through the prop itself so its more explicit whats determining the changes rather than adding another layer of abstraction.

rightAlign because isOnwardContent
feels clearer to me than
rightAlign because roundedImage